tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -90,6 +90,20 @@ public static function rebootKernelPrepareFeature()
         self::$kernel->reboot($warmupDir);
     }
 
+    /**
+     * This hook can be used to flag a feature for kernel reboot, this is useful
+     * to force recreation of services (e.g: when you add some currencies in the
+     * database, you may need to reset the CLDR related services to use the new ones)
+     *
+     * @BeforeScenario  @reboot-kernel-before-scenario
+     */
+    public static function rebootKernelPrepareScenario()
```

Comment:
I guess the `reboot-kernel-before-feature` was not enough? In previous feature as long as the currencies were created in the background it worked